src/BadgeImageRendererService.php

```diff
@@ -36,7 +36,7 @@ public function __construct()
 
     public function renderBadgeWithCount(
         string $label,
-        int $count,
+        float $count,
```

Comment:
To prevent fatal errors if the sum of the base number and the count exceeds the maximum integer (gets casted to float).
At extremely large numbers this probably won't return accurate numbers since:
- As mentioned in an above comment, `baseNumber` gets converted to the maximum integer at extremely large numbers
- `number_format` returns inaccurate values at extremely large numbers

Reply:
Not sure we need to handle extremely large values. I'd suggest to render error "max value reached"